packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/telephony-line-phone-accessories.html

```diff
@@ -0,0 +1,36 @@
+<section class="telecom-telephony-line-phone-accessories">
+    <header>
+        <tuc-section-back-link
+            data-tuc-section-back-link-to-state="telecom.telephony.billingAccount.line.phone"
+            data-tuc-section-back-link-title="{{ 'telephony_group_line_phone_section_back_link' | translate }}"
+        >
+        </tuc-section-back-link>
+        <h1 data-translate="telephony_line_phone_accessories_title"></h1>
+    </header>
+
+    <div class="text-center" data-ng-if="$ctrl.loading.init">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <tuc-toast-message></tuc-toast-message>
+
+    <div data-ng-if="!$ctrl.loading.init" class="telecom-section-content">
+        <div data-ng-if="$ctrl.process.currentView === 'choice'">
```

Comment:
Why not use different state instead with a `ui-view` as root and have `/choice`, `/shipping`, `/resume` and `/finalize`?

Reply:
I just re-used the old version of the code (not rewrite all)